### ISSUE TITLE
Keep style as generated by QGIS

### ISSUE THREAD

**Issue by aaime** [boundlessgeo/qgis-geoserver-plugin#107]
The plugin is trying to alter the style by text processing, bringing it down to SLD 1.0.
Doing so is necessary only for old versions of GeoServer, and can reduce the amount of symbology effectively exported in latest versions.
Also, in QGIS 3.5 QGIS can generate raster styles on its own with a good variety of options, so it would be best to use the internal style.

At the very least, it would be good to have an option not to alter the QGIS generated style, and allow users to choose which approach they prefer.

**Comment by pvgenuchten** [comment]
I wondered indeed that 'save style as sld' provides a nice categorised sld, however the plugin created a very basic single-style sld. I almost feared all sld generation was re-implemented in the plugin...

**Comment by matself** [comment]
Is this plugin even available for QGIS 3? I do not think so. The current version, 0.6 seems available only for QGIS 2.18.

**Comment by pvgenuchten** [comment]
yes, this works fine in qgis3, but use master branch, no recent releases afaik

**Comment by matself** [comment]
Hi!
Then why not release it for QGIS 3?
But I doubt that will happen, given Boundless’s more recent business development. The plugin was crappy anyway. 
That said, a working QGIS-Geoserver connector would be the greatest gift to the OSGEO community. 

Hälsning / Regards
Mats.E

Skickat från min / Sent from my iPad, Ursäkta att jag är kortfattad / Excuse my brevity.
